assets/scss/components/stencil/heroCarousel/_heroCarousel.scss

```diff
@@ -75,8 +78,14 @@
         }
     }
 
+    &.heroCarousel-slide-stretch .heroCarousel-slide {
```

Comment:
If you want to modify the appearance of a CSS selector, you should use `--` instead of `-`. Also I think this is unneccessarily nested. You can lower its specificity by moving it out. i.e.:

```css
.heroCarousel-slide--stretched {
    @include breakpoint("large") {
        background-size: 100%;
    }
}
```

And apply the modifier class directly to `.heroCarousel-slide` :)